execution-env.md

```diff
@@ -449,13 +456,63 @@ Fusing and reassociation of floating point operations is allowed when those inst
 
 ### Instructions
 
+Supported OpCodes are listed in [Appendix A](#a-supported-opcodes).
 
+The following sections describe differences from specific SPIR-V instructions.
 
-*   OpUndef is not allowed.
-*   OpVectorShuffle may not have a component literal with value 0xFFFFFFFF.
+####  Miscellaneous instructions
+
+*   **OpUndef** is not allowed.
 
-Supported OpCodes are iterated in [Appendix A](#a-supported-opcodes).
+####  Constant creation instructions
+
+*   **OpSpecConstantOp**:
+   * Opcode **OpVectorShuffle** may not have a component literal with value 0xFFFFFFFF.
```

Comment:
It looks like the OpVectorShuffle bit was added in twice, and there doesn't appear to be any description for OpSpecConstantOp

Reply:
Talked with dneto@ about this in person. This correct, since it is describing the same effective change in two different parts of the spec.

Reply:
This first case is for when OpSpecConstantOp is used as the Opcode operand to an OpSpecConstantOp constant-creation instruction (in the types-and-constants section of the module).  The next case in the next section is when OpVectorShuffle is used as its own standalone executable instruction (i.e. inside a function body.)